src/lambda/snowplow/tracker.ts

```diff
@@ -0,0 +1,23 @@
+import {
+  gotEmitter,
+  HttpMethod,
+  HttpProtocol,
+  tracker as snowPlowTracker,
+} from '@snowplow/node-tracker';
+import config from '../config';
+
+const emitter = gotEmitter(
+  config.snowplow.endpoint,
+  config.snowplow.httpProtocol as HttpProtocol,
+  undefined,
+  HttpMethod.POST,
+  config.snowplow.bufferSize,
+  config.snowplow.retries
+);
+
+export const tracker = snowPlowTracker(
```

Comment:
Curious -- do we ever need `tracker` alone? Or should this just be an implementation detail of `EventHandler` and combined for simplicty

Reply:
ya most of the `snowplow` code/integration test was a copy-paste from the user-api...kept the goal to port the code here and tie it up with event bridge

Reply:
revisitng this, if we have `tracker` inside the event-handler class, we need to inject this as we are using the same `tracker` for integration test